Reports/router.js

```diff
@@ -1,4 +1,38 @@
 var express = require('express');
+var mongoose = require('mongoose');
```

Comment:
@jmtaber129 I agree with @the-pat on separating the logic out. It's good for now but once things get a little more complicated with things such as authentication and error handling, that future PR you reference will be needed. I am a big believer in the [Single Responsibility Principle](https://www.toptal.com/software/single-responsibility-principle)